backend/src/test/java/codezap/category/service/facade/MemberCategoryApplicationServiceTest.java

```diff
@@ -0,0 +1,180 @@
+
+            FindAllCategoriesResponse response = sut.findAllByMember(member.getId());
+
+            assertThat(response.categories()).hasSize(2)
+                    .containsExactly(FindCategoryResponse.from(category1), FindCategoryResponse.from(category2))
+                    .doesNotContain(FindCategoryResponse.from(category3));
+        }
+
+        @Test
+        @DisplayName("멤버에 따른 카테고리 조회 실패 : 멤버 존재하지 않음")
+        void findAllByMemberFailNotExistsMember() {
+            Long notExistsId = 100L;
+
+            assertThatThrownBy(() -> sut.findAllByMember(notExistsId))
+                    .isInstanceOf(CodeZapException.class)
+                    .hasMessage("식별자 " + notExistsId + "에 해당하는 멤버가 존재하지 않습니다.");
+        }
+    }
+
+    @Nested
+    @DisplayName("카테고리 수정")
+    class Update {
+        @Test
+        @DisplayName("카테고리 수정 성공")
+        void updateSuccess() {
+            Member member = memberRepository.save(MemberFixture.memberFixture());
+            MemberDto memberDto = MemberDto.from(member);
+            Category category = categoryRepository.save(new Category("카테고리 1", member));
+            String renameCategoryName = "카테고리 수정";
+            UpdateCategoryRequest updateCategoryRequest = new UpdateCategoryRequest(renameCategoryName);
+
+            sut.update(memberDto, category.getId(), updateCategoryRequest);
+
+            Category actual = categoryRepository.fetchById(category.getId());
+            assertAll(
+                    () -> assertThat(actual).isEqualTo(category),
+                    () -> assertThat(actual.getName()).isEqualTo(renameCategoryName)
+            );
+        }
+
+        @Test
+        @DisplayName("카테고리 수정 실패 : 멤버 존재하지 않음")
+        void updateFailNotExistsMember() {
+            MemberDto memberDto = MemberDto.from(MemberFixture.memberFixture());
+            String renameCategoryName = "카테고리 수정";
+            UpdateCategoryRequest updateCategoryRequest = new UpdateCategoryRequest(renameCategoryName);
+
+            assertThatThrownBy(() -> sut.update(memberDto, 1L, updateCategoryRequest))
+                    .isInstanceOf(CodeZapException.class)
+                    .hasMessage("식별자 " + memberDto.id() + "에 해당하는 멤버가 존재하지 않습니다.");
+        }
+
+        @Test
+        @DisplayName("카테고리 수정 실패 : 카테고리 이름 중복")
+        void updateFailDuplicatedCategory() {
+            Member member = memberRepository.save(MemberFixture.memberFixture());
+            MemberDto memberDto = MemberDto.from(member);
+            Category category = categoryRepository.save(new Category("카테고리 1", member));
+            categoryRepository.save(new Category("카테고리 2", member));
+            String renameCategoryName = "카테고리 2";
+            UpdateCategoryRequest updateCategoryRequest = new UpdateCategoryRequest(renameCategoryName);
+            Long notExistsId = category.getId() + 1;
+
+            assertThatThrownBy(() -> sut.update(memberDto, notExistsId, updateCategoryRequest))
```

Comment:
놓치지 않도록 이곳에서도 동일 내용 표시해 둘게요!
```suggestion
            assertThatThrownBy(() -> sut.update(MemberDto.from(member), notExistsId, updateCategoryRequest))
```